automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Table.scala

```diff
@@ -40,23 +40,23 @@ case class Table(private val id: String)(implicit webDriver: WebDriver)
   }
 
   def goToPreviousPage(): Unit = {
+    prevPageButton.doClick()
```

Comment:
is there a reason to move away from the "click on x" to "x.doClick()"?
we still use "click on" in other places in this file

Reply:
I'm having issues getting inner components working. So for right now, we have to use raw `Query`s for nested components. I'll make this more consistent.